xml/System.ServiceModel.Federation/WSTrustChannelSecurityTokenProvider.xml

```diff
@@ -1,887 +1,887 @@
+      <MemberSignature Language="DocId" Value="E:System.ServiceModel.Federation.WSTrustChannelSecurityTokenProvider.System#ServiceModel#ICommunicationObject#Closed" />
+      <MemberSignature Language="VB.NET" Value="Custom Event Closed As EventHandler Implements System.ServiceModel.ICommunicationObject" />
+      <MemberSignature Language="F#" Value="member this.System.ServiceModel.ICommunicationObject.Closed : EventHandler " Usage="member this.System.ServiceModel.ICommunicationObject.Closed : System.EventHandler " />
+      <MemberType>Event</MemberType>
+      <Implements>
+        <InterfaceMember>E:System.ServiceModel.ICommunicationObject.Closed</InterfaceMember>
+      </Implements>
+      <AssemblyInfo>
+        <AssemblyName>System.ServiceModel.Federation</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.EventHandler</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Occurs when the communication object completes its transition from the closing state into the closed state.</summary>
+        <remarks>To be added.</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="System.ServiceModel.ICommunicationObject.Closing">
+      <MemberSignature Language="C#" Value="event EventHandler System.ServiceModel.ICommunicationObject.Closing;" />
+      <MemberSignature Language="ILAsm" Value=".event class System.EventHandler System.ServiceModel.ICommunicationObject.Closing" />
+      <MemberSignature Language="DocId" Value="E:System.ServiceModel.Federation.WSTrustChannelSecurityTokenProvider.System#ServiceModel#ICommunicationObject#Closing" />
+      <MemberSignature Language="VB.NET" Value="Custom Event Closing As EventHandler Implements System.ServiceModel.ICommunicationObject" />
+      <MemberSignature Language="F#" Value="member this.System.ServiceModel.ICommunicationObject.Closing : EventHandler " Usage="member this.System.ServiceModel.ICommunicationObject.Closing : System.EventHandler " />
+      <MemberType>Event</MemberType>
+      <Implements>
+        <InterfaceMember>E:System.ServiceModel.ICommunicationObject.Closing</InterfaceMember>
+      </Implements>
+      <AssemblyInfo>
+        <AssemblyName>System.ServiceModel.Federation</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.EventHandler</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Occurs when the communication object first enters the closing state.</summary>
+        <remarks>This member is an explicit interface member implementation. It can be used only when the &lt;xref:System.ServiceModel.Federation.WSTrustChannelSecurityTokenProvider&gt; instance is cast to an &lt;xref:System.ServiceModel.ICommunicationObject&gt; interface.</remarks>

```

Comment:
```suggestion
        <remarks>This member is an explicit interface member implementation. It can be used only when the <see cref="T:System.ServiceModel.Federation.WSTrustChannelSecurityTokenProvider"/> instance is cast to an <see cref="T:System.ServiceModel.ICommunicationObject"/> interface.</remarks>

```